Library/Homebrew/brew.sh

```diff
@@ -127,6 +127,17 @@ then
   HOMEBREW_CACHE="${HOMEBREW_CACHE:-${HOME}/Library/Caches/Homebrew}"
   HOMEBREW_LOGS="${HOMEBREW_LOGS:-${HOME}/Library/Logs/Homebrew}"
   HOMEBREW_SYSTEM_TEMP="/private/tmp"
+
+  # Set a variable when the macOS system Ruby is new enough to avoid spawning
+  # a Ruby process unnecessarily.
+  if [[ "$HOMEBREW_MACOS_VERSION_NUMERIC" -lt "101303" ]]
+  then
+    unset HOMEBREW_MACOS_SYSTEM_RUBY_NEW_ENOUGH
+  else
+    # Used in ruby.sh.
+    # shellcheck disable=SC2034
+    HOMEBREW_MACOS_SYSTEM_RUBY_NEW_ENOUGH="1"
```

Comment:
Can you think of a clearer name? Maybe `HOMEBREW_MACOS_SYSTEM_RUBY_VERSION_CHECK=0`.

That would say to me: "if the macOS numeric version is greater than X, don't check the ruby version".

Reply:
I think this name is clearer than the suggestion and cannot think of a better one.